commit 9c38ee2af9eef411b0795ccdae30bd7da730fba2
Author: GANGA BHAVANI YENUGULA <gyenugul@qti.qualcomm.com>
Date:   Thu Sep 25 11:26:22 2025 +0530

    This patch resolves a build error in pam_namespace.c
    due to 'LC_COLLATE' being undeclared.
    The issue was caused by the removal of <locale.h>
    from pam_namespace.h in CVE-2025-6020-01.patch.

    Reintroducing the header ensures setlocale() functions
    compile correctly in Linux-PAM 1.5.2.

sign-off-by: GANGA BHAVANI YENUGULA <gyenugul@qti.qualcomm.com>

diff --git a/modules/pam_namespace/pam_namespace.h b/modules/pam_namespace/pam_namespace.h
index 8a7a66bd..071ca57a 100644
--- a/modules/pam_namespace/pam_namespace.h
+++ b/modules/pam_namespace/pam_namespace.h
@@ -55,6 +55,7 @@
 #include <fcntl.h>
 #include <sched.h>
 #include <glob.h>
+#include <locale.h>
 #include "security/pam_modules.h"
 #include "security/pam_modutil.h"
 #include "security/pam_ext.h"
